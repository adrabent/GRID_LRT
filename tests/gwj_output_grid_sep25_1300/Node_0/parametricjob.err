+ tar -xf picas.tar
+ tar -xf couchdb.tar
+ python getOBSID.py spectroscopy raymond reeJua2f
Traceback (most recent call last):
  File "getOBSID.py", line 67, in <module>
    main()
  File "getOBSID.py", line 64, in main
    actor.run()
  File "/data/scratch/5583283.batch.gina.sara.nl/CREAM699215107/picas/actors.py", line 28, in run
    for key, ref, token in self.iterator:            
  File "/data/scratch/5583283.batch.gina.sara.nl/CREAM699215107/picas/iterators.py", line 32, in next
    return self.claim_token()
  File "/data/scratch/5583283.batch.gina.sara.nl/CREAM699215107/picas/iterators.py", line 64, in claim_token
    view_params=self.view_params, window_size=100)
  File "/data/scratch/5583283.batch.gina.sara.nl/CREAM699215107/picas/clients.py", line 51, in get_token
    l = len(view.rows)
  File "/data/scratch/5583283.batch.gina.sara.nl/CREAM699215107/couchdb/client.py", line 1176, in rows
    self._fetch()
  File "/data/scratch/5583283.batch.gina.sara.nl/CREAM699215107/couchdb/client.py", line 1163, in _fetch
    data = self.view._exec(self.options)
  File "/data/scratch/5583283.batch.gina.sara.nl/CREAM699215107/couchdb/client.py", line 1029, in _exec
    _, _, data = _call_viewlike(self.resource, options)
  File "/data/scratch/5583283.batch.gina.sara.nl/CREAM699215107/couchdb/client.py", line 1088, in _call_viewlike
    return resource.get_json(**_encode_view_options(options))
  File "/data/scratch/5583283.batch.gina.sara.nl/CREAM699215107/couchdb/http.py", line 523, in get_json
    return self._request_json('GET', path, headers=headers, **params)
  File "/data/scratch/5583283.batch.gina.sara.nl/CREAM699215107/couchdb/http.py", line 546, in _request_json
    headers=headers, **params)
  File "/data/scratch/5583283.batch.gina.sara.nl/CREAM699215107/couchdb/http.py", line 542, in _request
    credentials=self.credentials)
  File "/data/scratch/5583283.batch.gina.sara.nl/CREAM699215107/couchdb/http.py", line 392, in request
    raise ResourceNotFound(error)
couchdb.http.ResourceNotFound: (u'not_found', u'missing')
+ echo 'Pulling down the sandbox for OBSID  from /pnfs/grid.sara.nl/data/lofar/user/disk/spectroscopy/sandbox/sandbox_raymond_.tar'
+ uberftp -rm gsiftp://gridftp.grid.sara.nl:2811/pnfs/grid.sara.nl/data/lofar/user/sksp/spectroscopy-migrated/sandbox/sandbox_spectroscopy_.tar
No match for /pnfs/grid.sara.nl/data/lofar/user/sksp/spectroscopy-migrated/sandbox/sandbox_spectroscopy_.tar
+ ls -l
