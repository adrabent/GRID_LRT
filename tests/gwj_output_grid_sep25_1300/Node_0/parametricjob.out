Pulling down the sandbox for OBSID  from /pnfs/grid.sara.nl/data/lofar/user/disk/spectroscopy/sandbox/sandbox_raymond_.tar
total 276
drwxr-xr-x 4 lfr027 lofar   4096 Sep 25 12:47 couchdb
-rw-r--r-- 1 lfr027 lofar 225280 Sep 25 12:47 couchdb.tar
-rw-r--r-- 1 lfr027 lofar   2635 Sep 25 12:47 getOBSID.py
-rw-r--r-- 1 lfr027 lofar   2451 Sep 25 12:47 parametricjob.err
-rw-r--r-- 1 lfr027 lofar    123 Sep 25 12:47 parametricjob.out
drwxr-xr-x 2 lfr027 lofar   4096 Sep 25 12:47 picas
-rw-r--r-- 1 lfr027 lofar  30720 Sep 25 12:47 picas.tar
-rw-r--r-- 1 lfr027 lofar   1284 Sep 25 12:47 run_remote_sandbox.sh
